indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -256,3 +259,31 @@ def _process(self):
         editors = get_editors(self.event, self.editable_type)
         return jsonify_template('events/editing/management/editor_list.html',
                                 event_persons=editors, event=self.event)
+
+
+class RHEmailNotSubmittedEditablesMetadata(EmailRolesMetadataMixin, RHEditableTypeManagementBase):
+    object_context = 'contributions'
+
+
+class RHEmailNotSubmittedEditablesPreview(EmailRolesPreviewMixin, RHEditableTypeManagementBase):
+    object_context = 'contributions'
+
+    def get_placeholder_kwargs(self):
+        contribution = Contribution.query.with_parent(self.event).first()
```

Comment:
I think there should be some check in case there are no contribs in the event. Otherwise the email preview breaks when you use any of the contribution placeholders

Reply:
Fixed, and also added a count of contributions w/o editables

Reply:
I like it!